### PR TITLE
feat: update forum template

### PR DESCRIPTION
**Pull request by nicosantangelo**
Example of the new template:
https://forum.decentraland.org/t/a-new-wearables-collection-created-by-new-address-who-dis-is-ready-for-review/635

Closes https://github.com/decentraland/builder-server/issues/200

### CODE REVIEW

File: src/modules/wallet/sagas.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { MANA_ADDRESS } from 'modules/common/contracts'
 
 const baseWalletSaga = createWalletSaga({
   MANA_ADDRESS,
-  CHAIN_ID: env.get('REACT_APP_CHAIN_ID') || ChainId.ETHEREUM_MAINNET
+  CHAIN_ID: env.get('REACT_APP_CHAIN_ID') || ChainId.ETHEREUM_GOERLI
```
Comment (nachomazzara) [inline src/modules/wallet/sagas.ts:20]: Is this necessary?

**Review by nachomazzara — COMMENTED** [review]
LGTM!

File: src/modules/wallet/sagas.ts (hunk 1 above)
Comment (nicosantangelo) [reply src/modules/wallet/sagas.ts:20]: it is not!

**Comment by nicosantangelo** [comment]
Updated!

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**